Packs/ForescoutEyeInspect/IncidentFields/incidentfield-ForescoutEyeInspect_L3_Protocol.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
@RazOmry We are trying to minimize the creation of new fields. Please considerusing the field "Protocol" instead. I didn't comment on the L2 protocol since we don't have a separation for L2 and L3 in the common fields at the moment.